spec/abilities/collection_ability_spec.rb

```diff
@@ -321,7 +321,7 @@
              permission_template: collection.permission_template,
              agent_type: 'user',
              agent_id: user.user_key)
-      collection.reset_access_controls!
+      collection.permission_template.reset_access_controls_for(collection:)
```

Comment:
Rhetorical/Venting: Upstream change brings the Yuck.  Shouldn't the collection be able to do this?

I mean if it gets things working.  But saying `A` tells `B` to do `C` with `A` is a very circuitous way; but